system-test/spanner.js

```diff
@@ -3070,6 +3070,34 @@ describe('Spanner', function() {
     });
 
     describe('read/write', function() {
+      it('should throw an error for mismatched columns', function(done) {
```

Comment:
So this test would fail without the fix introduced in this PR?

Reply:
Correct, it would return a rejected promise instead of throwing an error.